interfaces/builtin/system_packages_doc_test.go

```diff
@@ -106,13 +110,16 @@ func (s *systemPackagesDocSuite) TestMountSpec(c *C) {
 	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, s.coreSlot), IsNil)
 
 	entries := spec.MountEntries()
-	c.Assert(entries, HasLen, 2)
+	c.Assert(entries, HasLen, 3)
 	c.Check(entries[0].Name, Equals, "/var/lib/snapd/hostfs/usr/share/doc")
 	c.Check(entries[0].Dir, Equals, "/usr/share/doc")
 	c.Check(entries[0].Options, DeepEquals, []string{"bind", "ro"})
 	c.Check(entries[1].Name, Equals, "/var/lib/snapd/hostfs/usr/share/libreoffice/help")
 	c.Check(entries[1].Dir, Equals, "/usr/share/libreoffice/help")
 	c.Check(entries[1].Options, DeepEquals, []string{"bind", "ro"})
+	c.Check(entries[0].Name, Equals, "/var/lib/snapd/hostfs/usr/share/xubuntu-docs")
+	c.Check(entries[0].Dir, Equals, "/usr/share/xubuntu-docs")
+	c.Check(entries[0].Options, DeepEquals, []string{"bind", "ro"})
```

Comment:
These should be `entries[2]`.

Reply:
Of course. That's a dumb copy-paste error. Thanks for spotting it!